### PR TITLE
pskubectlcompletion: Add version 1.0.4

### PR DESCRIPTION
**Pull request by alexchapar**
This can be use to enable kubectl auto-completion for PowerShell

### CODE REVIEW

File: bucket/pskubectlcompletion.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "version": "1.0.4",
+    "description": "kubectl auto-completion for PowerShell.",
+    "homepage": "https://github.com/mziyabo/PSKubectlCompletion/",
+    "license": "Apache-2.0",
+    "notes": [
+        "Use the module by running: 'Import-Module PSKubectlCompletion'",
+        "Add that command to your $PROFILE to make it permanent",
+        "",
+        "See usage: https://github.com/mziyabo/PSKubectlCompletion#powershell-auto-completion-for-kubectl"
+    ],
```
Comment (Ash258) [inline bucket/pskubectlcompletion.json:11]: The notes are pretty much irrelevant

```suggestion
```

File: bucket/pskubectlcompletion.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    "hash": "497d7c0724b5946176d8a6a250e074d1bf3848e9e75adaf2d73d623f57d1716e",
+    "psmodule": {
+        "name": "PSKubectlCompletion"
+    },
+    "checkver": {
+        "url": "https://www.powershellgallery.com/packages/PSKubectlCompletion",
+        "regex": "<h2>([\\d.]+)</h2>"
+    },
+    "autoupdate": {
+        "url": "https://psg-prod-eastus.azureedge.net/packages/pskubectlcompletion.$version.nupkg#/dl.7z"
```
Comment (Ash258) [inline bucket/pskubectlcompletion.json:22]:
```suggestion
        "url": "https://psg-prod-eastus.azureedge.net/packages/pskubectlcompletion.$version.nupkg"
```

File: bucket/pskubectlcompletion.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{
+    "version": "1.0.4",
+    "description": "kubectl auto-completion for PowerShell.",
+    "homepage": "https://github.com/mziyabo/PSKubectlCompletion/",
```
Comment (Ash258) [inline bucket/pskubectlcompletion.json:4]:
```suggestion
    "description": "kubectl auto-completion for PowerShell",
    "homepage": "https://github.com/mziyabo/PSKubectlCompletion",
```

File: bucket/pskubectlcompletion.json [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    "license": "Apache-2.0",
+    "notes": [
+        "Use the module by running: 'Import-Module PSKubectlCompletion'",
+        "Add that command to your $PROFILE to make it permanent",
+        "",
+        "See usage: https://github.com/mziyabo/PSKubectlCompletion#powershell-auto-completion-for-kubectl"
+    ],
+    "url": "https://psg-prod-eastus.azureedge.net/packages/pskubectlcompletion.1.0.4.nupkg#/dl.7z",
+    "hash": "497d7c0724b5946176d8a6a250e074d1bf3848e9e75adaf2d73d623f57d1716e",
```
Comment (Ash258) [inline bucket/pskubectlcompletion.json:13]:
```suggestion
    "hash": "497d7c0724b5946176d8a6a250e074d1bf3848e9e75adaf2d73d623f57d1716e",
    "pre_install": "Remove-Item \"$dir\\_rels\", \"$dir\\package\", \"$dir\\*Content*.xml\" -Recurse",
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

**Comment by alexchapar** [comment]
Thanks for the comments.. I've applied them

**Review by Ash258 — APPROVED** [review]

**Pull request merged**